spec/controllers/datasets_controller_spec.rb

```diff
@@ -1,68 +0,0 @@
-require 'rails_helper'
-
-describe DatasetsController, type: :controller do
```

Comment:
This behaviour being tested here is unrelated to signon. We should continue to test it.

Reply:
This controller test was testing authorisation logic, which I've removed as it's incompatible with the new user model (also half-baked). The tests needed to go as the logic they tested is also gone.